lib/manageiq/providers/embedded_terraform/engine.rb

```diff
@@ -18,9 +18,9 @@ def self.vmdb_plugin?
         def self.menu
           [
             Menu::Section.new(:embedded_terraform_automation_manager, N_("Embedded Terraform"), nil, [
-              Menu::Item.new('embedded_terraform_template',    N_('Templates'),    'embedded_terraform_template',    {:feature => 'embedded_configuration_script_payload_view', :any => true}, '/embedded_terraform_template/show_list'),
-              Menu::Item.new('embedded_terraform_repository',  N_('Repositories'), 'embedded_terraform_repository',  {:feature => 'embedded_configuration_script_source_view',  :any => true}, '/embedded_terraform_repository/show_list'),
-              Menu::Item.new('embedded_terraform_credentials', N_('Credentials'),  'embedded_terraform_credentials', {:feature => 'embedded_configuration_script_source_view',  :any => true}, '/embedded_terraform_credential/show_list')
```

Comment:
This fixes the menu sections to reuse the existing shared product feature.  This means that if you enable either embedded ansible or embedded terraform features, they are shared so, the check box for one will be enable it for the other.

![image](https://github.com/ManageIQ/manageiq-providers-embedded_terraform/assets/19339/9a77dcd6-caf2-4266-aea0-4635c3b295a8)

For example, I enabled just configuration scripts in embedded ansible and it enables it for embedded terraform after saving and reloading the page.